### ISSUE TITLE
Rethink color system

### ISSUE THREAD

**Issue by valadas** [DNNCommunity/dnn-elements#728]
We need to add some colors for danger, warning, success, neutrals

**Comment by david-poindexter** [comment]
Here is where we landed (as of Jun 18, 2023).

```css
--primary
--primary-light
--primary-dark
--primary-contrast

--secondary
--secondary-light
--secondary-dark
--secondary-contrast

--tertiary
--tertiary-light
--tertiary-dark
--tertiary-contrast

--neutral
--neutral-light
--neutral-dark
--neutral-contrast

--background
--background-light
--background-dark
--background-contrast

--foreground
--foreground-light
--foreground-dark
--foreground-contrast

--info
--info-light
--info-dark
--info-contrast

--success
--success-light
--success-dark
--success-contrast

--warning
--warning-light
--warning-dark
--warning-contrast

--danger
--danger-light
--danger-dark
--danger-contrast
```

**Comment by valadas** [comment]
Merged into `release/0.19.0`
